src/it/resources-noncompilable/org/checkstyle/checks/imports/importorder/InputFromAndroid.java

```diff
@@ -0,0 +1,55 @@
+//non-compiled with javac: contains specially crafted set of imports for testing
+package org.checkstyle.checks.importorder;
+/**
+ * This test-input is intended to be checked using following configuration:
+ *
+ * groups = {android,androidx,com.android,dalvik,com,gov,junit,libcore,net,org,java,javax}
+ * option = top
+ * ordered = true
+ * separated = true
+ *
+ * This test-input is based on real application code and shouldn't be changed.
+ * @see https://android.googlesource.com/platform/prebuilts/checkstyle/+/master/default-treewalker-checks.xml
+ */
+import static android.graphics.drawable.Icon.TYPE_ADAPTIVE_BITMAP;
+import static android.graphics.drawable.Icon.TYPE_BITMAP;
+import static android.graphics.drawable.Icon.TYPE_DATA;
+import static android.graphics.drawable.Icon.TYPE_RESOURCE;
+import static android.graphics.drawable.Icon.TYPE_URI;
+
+import static androidx.annotation.RestrictTo.Scope.LIBRARY; // warn
```

Comment:
@pbludov ,can you explain this warn ?
Intention of such IT was to have no violations on them as in real code.

If additional changes are required to configuration - we need to make them and warn users that on upgrade they need to upgrade their configs.

looks like problem is due fact that before 8.3 extra spaces between imports were ignored. Now it is a problem.
So users have to use `groupsSeparated={SAME AS GROUPS}` and `separatedStatic=true` to have no violations.

Am I right ?

If yes - please update IT Inputs to have no violations.

Reply:
>So users have to use groupsSeparated={SAME AS GROUPS} and separatedStatic=true to have no violations.

Indeed.

>If yes - please update IT Inputs to have no violations.

done